docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc

```diff
@@ -1,250 +1,333 @@
-| Fields | _(blank)_ | A comma or space separated list of fields to show for an annotation list or table. The given field order is reflected in the view.
-| Annotations | (checked) |  Whether or not to show annotations for an alert table.
 |====================
 
-
 [[monitoring-console-widget-decorations]]
-==== Configuring Widget Decorations
-Decorations are visual helpers that can be added to a graph of a widget in order to make it easier to for the user to quickly understand the data displayed. In particular these are reference lines.
+[[monitoring-console-widget-extras]]
+==== Configuring Widget Extras
 
-image::monitoring-console/mc_settings_decorations.png[Widget Decoration Settings]
+The _Extras_ settings are concerned with additional information shown in or around the graph or widget content in general.
+This includes visual helpers that can be added to a graph of a widget in order to make it easier to for the user to quickly understand the data displayed.
+In particular these are reference lines and threshold indications.
 
+image::monitoring-console/mc_settings_widget_extras-h.png[Widget Extras Settings]
+
+.Widget Extras Settings
 [cols="1,1,4",options="header"]
 |====================
 | Property | Default | Description
+| Annotations | (checked) |  Whether or not to show annotations for in an alert table.
+| Annotations (Fields) | _(blank)_ | A comma or space separated list of fields to show for an annotation list or table. The given field order is reflected in the view.
+| Aggregates | (none checked) | Check to add the all time minimum, maximum or average line to the graph
 | Waterline | (undefined) | Adds a simple reference line that could mark a upper or lower limit. Enter in unit of y-axis, e.g. 20ms, 5s, 2GB, 4%, 123
 | Threshold Reference | `Off` | Controls what value the _Alarming Threshold_ and _Critical Threshold_ are compared to. Usually this is the most recent value but in some cases the all time minimum, maximum or average are useful too.
 | Alarming Threshold | (undefined) | The limit for the _"Alarming"_ state. If _Critical Threshold_ is undefined or if it's value is larger than this value the alarming threshold is exceeded if the reference value is larger than the threshold. Otherwise, if the _Critical Threshold_ is lower than the _Alarming Threshold_, the threshold becomes 'inverse', and is exceeded when the reference value falls below the alarming value. When exceeded the status becomes _"Alarming"_. Enter in unit of y-axis, e.g. 20ms, 5s, 2GB, 4%, 123
```

Comment:
```suggestion
| Alarming Threshold | (undefined) | The limit for the _"Alarming"_ state. If _Critical Threshold_ is undefined or if its value is larger than this value the alarming threshold is exceeded if the reference value is larger than the threshold. Otherwise, if the _Critical Threshold_ is lower than the _Alarming Threshold_, the threshold becomes 'inverse', and is exceeded when the reference value falls below the alarming value. When exceeded the status becomes _"Alarming"_. Enter in unit of y-axis, e.g. 20ms, 5s, 2GB, 4%, 123
```